conan/tools/cmake/cmake.py

```diff
@@ -111,6 +111,20 @@ def configure(self, variables=None, build_script_folder=None, cli_args=None):
         with chdir(self, build_folder):
             self._conanfile.run(command)
 
+    @property
+    def _verbosity(self):
+        verbosity = self._conanfile.conf.get("tools.build.verbosity")
```

Comment:
```suggestion
        verbosity = self._conanfile.conf.get("tools.build:verbosity")
```

Reply:
Omg 🤦 I ctrl-f to replace all the occurrences, and still this fell thru the cracks